### ISSUE TITLE
Feature: Setting estimate on task activation

### ISSUE THREAD

**Issue by pstibrany** [Tasktop/Foglyn#29]
It would be great to be able to set estimate for case which doesn't have one, but is activated in Mylyn.

UI should be non-obtrusive, but warn user that this task was not activated in FogBugz and he should set estimate first.

**Comment by elklaaso** [comment]
I totally agree, another reaon why sometimes by time on a task does not get tracked, and i invoice my client not enough hours.
